### DISCUSSION TITLE
Can anyone tell me how to get information about consumer pending metrics on  nats-surveyor or prometheus-nats-exporter

### DISCUSSION THREAD

**Discussion by Doslin** [nats-io/nats-server#3857] [Q&A]
![image](https://user-images.githubusercontent.com/29807388/217513721-d691da61-24a1-44f3-9041-ea37084cf0b6.png)

**Comment by derekcollison** [comment]
That is the number of messages still to be delivered to the consumer, with consideration for any filter subject provided.

**Reply by Doslin** [reply]
Are there any metrics in nats-surveyor or prometheus-nats-exporter?

**Reply by derekcollison** [reply]
We have updates to surveyor that add JetStream but not sure that metric is there. /cc @ColinSullivan1
